kagiso_auth/models.py

```diff
@@ -48,6 +48,20 @@ def username(self):
     def username(self, value):
         self.email = value
 
+        # http://stackoverflow.com/a/9754466
+
+    # http://stackoverflow.com/a/9754466
+    @property
+    def age(self):
+        if self.profile and 'birth_date' in self.profile:
+            birth_date = parser.parse(self.profile['birth_date'])
+
+            today = timezone.now()
+            birthday_has_passed = (today.month, today.day) < (birth_date.month, birth_date.day)  # noqa
+
+            # considering that int(True) is 1 and int(False) is 0
+            return today.year - birth_date.year - birthday_has_passed
+
```

Comment:
Would it not be simpler to use something like 'datediff' and then just taking the year part?